lib/cork/text_wrapper.rb

```diff
@@ -41,17 +41,11 @@ def wrap_formatted_text(string, indent = 0, max_width = 80)
     # @param  [Fixnum] indent
     #         The number of spaces to insert before the string.
     #
-    # @param  [Fixnum] max_width
-    #         The maximum width to use to format the string if the terminal
+    # @param  [Fixnum] width
+    #         The width to use to format the string if the terminal
```

Comment:
`The width to use to format the string if the terminal` -> `The width to use when formatting the string in the terminal`